static/app/views/starfish/views/spans/selectors/domainSelector.spec.tsx

```diff
@@ -108,7 +108,7 @@ describe('DomainSelector', function () {
 
     await tick();
```

Comment:
you didn't put this here but i'd try to remove the tick and see if it still passes, findBy is preferred over manual ticks like you have